lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -0,0 +1,15 @@
+# Process-related anti-patterns
+
+This document outlines anti-patterns related to processes and process-based abstractions.
+
+## Code organization by process
+
+TODO.
+
+## Agent Obsession
+
+TODO (note: it also applies to GenServer, so we should generalize it)
```

Comment:
@lucasvegi this obsession also applies to GenServer, so we plan to generalize it. The idea is that you want to encapsulate your interactions with processes. We thought about calling it "Unencapsulated processes" but it may be too vague. Suggestions?

Reply:
@josevalim How about "Scattered processes"?

Reply:
Maybe "Scattered process interfaces"?

Reply:
Sounds much better to me!

Reply:
Lovely, great job with the name @josevalim